modules/core/src/main/java/org/locationtech/jts/coverage/CoverageSimplifier.java

```diff
@@ -64,7 +67,37 @@ public static Geometry[] simplify(Geometry[] coverage, double tolerance) {
     CoverageSimplifier simplifier = new CoverageSimplifier(coverage);
     return simplifier.simplify(tolerance);
   }
-  
+
+  /**
+   * Simplifies the boundaries of a set of polygonal geometries forming a coverage,
+   * preserving the coverage topology.
+   *
+   * @param coverage a set of polygonal geometries forming a coverage
+   * @param tolerances the simplification tolerances for each coverage
+   * @return the simplified polygons
+   */
+  public static Geometry[] simplify(Geometry[] coverage, List<Double> tolerances) {
+    CoverageSimplifier simplifier = new CoverageSimplifier(coverage);
+    return simplifier.simplify(tolerances);
+  }
+
+  /**
+   * Simplifies the boundaries of a set of polygonal geometries forming a coverage,
+   * preserving the coverage topology.
+   *
+   * @param coverage a set of polygonal geometries forming a coverage
+   * @param tolerances comma-separated string of simplification tolerances for each coverage
+   * @return the simplified polygons
+   */
+  public static Geometry[] simplify(Geometry[] coverage, String tolerances) {
```

Comment:
Conversion from a string to an array of tolerances should be left up to client code, since it's highly usage-specific.  It can be done in `CoverageFunctions` as well, to allow for easier testing and experimentation.

Reply:
Ok, then I will be moving that method into the JTS test builder code